### PR TITLE
add tabindex to coursses

### PR DESCRIPTION
**Pull request by Osman190**

### CODE REVIEW

File: controllers/UserController.js [hunk 1]
```diff
@@ -61,9 +61,10 @@ module.exports.register = async (req, res, next) => {
         token: userToken
       });
       User.createUser(newUser, async (err, user) => {
+        console.log(err)
         if (err) throw err;
         try {
-          await sendVerificationMail(res, req, userToken);
+          // await sendVerificationMail(res, req, userToken);
```
Comment (spielhoelle) [inline controllers/UserController.js:67]: We need this line

File: controllers/UserController.js [hunk 2]
```diff
@@ -61,9 +61,10 @@ module.exports.register = async (req, res, next) => {
         token: userToken
       });
       User.createUser(newUser, async (err, user) => {
+        console.log(err)
```
Comment (spielhoelle) [inline controllers/UserController.js:64]: I think we don't need that console.log here. But I don't know if we catch that thrown error in the next line neither.

File: views/courses.pug [hunk 3]
```diff
@@ -4,10 +4,10 @@ block content
     .container.h-100
       .h-100.py-3.py-md-5
         .row.py-5.justify-content-center
-          each course in courses
+          each course, index in courses
```
Comment (spielhoelle) [inline views/courses.pug:7]: We don't need that index

File: views/courses.pug [hunk 4]
```diff
@@ -17,4 +17,4 @@ block content
                   p.text-dark=course.subheading
                 div
                   hr
-                  a.text-dark.font-weight-bold(href="/courses/" + course.slug): +translate("View course")
+                  a.text-dark.font-weight-bold(tabindex="0" href="/courses/" + course.slug): +translate("View course")
```
Comment (spielhoelle) [inline views/courses.pug:20]: Tabindex 0 is default so also we don't need this?

**Comment by spielhoelle** [comment]
I think we should keep also the .keepme tule in the uploads.

**Review by spielhoelle — CHANGES_REQUESTED** [review]

File: views/courses.pug (hunk 4 above)
Comment (Osman190) [reply views/courses.pug:20]: but if I removed it it will not work, I tried it

File: controllers/UserController.js (hunk 2 above)
Comment (Osman190) [reply controllers/UserController.js:64]: can we force create user to throw an error? to check it it's really working

**Comment by Osman190** [comment]
what is this conflict at views/contactLocations.pug?

**Pull request closed without merging**